packages/js-sdk/src/tld/Tld.ts

```diff
@@ -48,6 +48,17 @@ export class Tld {
         this.userEnsSubdomain = userEnsSubdomain;
         this.resolverBackendUrl = resolverBackendUrl;
     }
+    isResolverForTldName(ensName: string): Promise<boolean> {
+        //Since its the root resolver its always capable of resolving
+        return Promise.resolve(true);
+    }
+    isResolverForAliasName(
+        ensName: string,
+        foreignTldName?: string,
+    ): Promise<boolean> {
+        //Since its the root resolver its always capable of resolving
+        return Promise.resolve(true);
```

Comment:
seems to be a redundant, what's is that used for?

Reply:
The interface isResolverForAliasName / isResolverForTldName are required by the ITLDResolver interface. It returns always true because the tld.ts Is the RootResolver and can always be used to retrieve a name because it knows every delegate.